notebooks/255-mms-massively-multilingual-speech/255-mms-massively-multilingual-speech.ipynb

```diff
@@ -0,0 +1,599 @@
+{
```

Comment:
<p>Does not sound right for me. May be we change it to <code>Do inference</code>? Or <code>Original model inference</code> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1257/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
Paraphrased